packages/core/src/login/webview/vue/toolkit/backend_toolkit.ts

```diff
@@ -14,12 +14,28 @@ import { CodeCatalystAuthenticationProvider } from '../../../../codecatalyst/aut
 export class ToolkitLoginWebview extends CommonAuthWebview {
     public override id: string = 'aws.toolkit.AmazonCommonAuth'
     public static sourcePath: string = 'vue/src/login/webview/vue/toolkit/index.js'
+    private isCodeCatalystEnterpriseLogin = false
 
     constructor(private readonly codeCatalystAuth: CodeCatalystAuthenticationProvider) {
         super(ToolkitLoginWebview.sourcePath)
     }
 
+    setLoginService(serviceToShow?: string) {
+        if (serviceToShow === 'codecatalyst') {
+            this.isCodeCatalystEnterpriseLogin = true
+        } else {
+            this.isCodeCatalystEnterpriseLogin = false
```

Comment:
the names are confusing. `setLoginService('codecatalyst')` sets a "enterprise" flag. why?

should `this.isCodeCatalystEnterpriseLogin` be named `this.isCodeCatalystLogin` ?